test/test.c

```diff
@@ -608,6 +608,8 @@ static rc_condset_t* trigger_get_set(rc_trigger_t* trigger, int ndx) {
 }
 
 static void test_trigger(void) {
+  char buffer[2048];
```

Comment:
fixes warning about possibly exceeding stack size. each test was apparently allocating its own 2K buffer on the stack